src/init.cpp

```diff
@@ -193,7 +193,7 @@ std::string HelpMessage()
     strUsage += "  -socks=<n>             " + _("Select the version of socks proxy to use (4-5, default: 5)") + "\n";
     strUsage += "  -tor=<ip:port>         " + _("Use proxy to reach tor hidden services (default: same as -proxy)") + "\n";
     strUsage += "  -dns                   " + _("Allow DNS lookups for -addnode, -seednode and -connect") + "\n";
-    strUsage += "  -port=<port>           " + _("Listen for connections on <port> (default: 15714 or testnet: 25714)") + "\n";
+    strUsage += "  -port=<port>           " + _("Listen for connections on <port> (default: 31174 or testnet: 31173)") + "\n";
```

Comment:
```
09:10 <@xploited> its depreciated. It wasn't removed from protocol.h though.  GetDefaultPort is used from chainparams 
09:10 <@xploited> not protocol.h
09:11 < dooglus> so that function should be deleted from protocol.h?
09:11 <@xploited> yeah
09:12 <@xploited> extern bool fTestNet;  as well,  thats not used anymore
```